install/operator/build.sh

```diff
@@ -36,13 +37,17 @@ where options are:
   --s2i-stream-name <name>                s2i image stream name (default: syndesis-operator)
   --go-options <name>                     additional build options to pass to the go build
   --go-proxy <url>                        proxy url for finding go dependencies (default: https://proxy.golang.org)
+  --productized <true|false>              activate settings only for product builds (default: false)
```

Comment:
This should not be necessary. We already have a productized flag in [config.yaml](https://github.com/syndesisio/syndesis/blob/master/install/operator/build/conf/config.yaml#L3). This is modified in the product version of that file.